src/collectors/perf.plugin/metadata.yaml

```diff
@@ -23,14 +23,14 @@ modules:
     overview:
       data_collection:
         metrics_description: "This collector monitors CPU performance metrics about cycles, instructions, migrations, cache operations and more."
-        method_description: "It uses syscall (2) to open a file descriptior to monitor the perf events."
+        method_description: "It uses syscall (2) to open a file descriptor to monitor the perf events."
       supported_platforms:
         include:
           - Linux
         exclude: []
       multi_instance: true
       additional_permissions:
-        description: "It needs setuid to use necessary syscall to collect perf events. Netada sets the permission during installation time."
+        description: "It needs setuid to use the necessary syscall to collect perf events. Netdada sets the permission during installation time."
```

Comment:
> Netdada

Netdata

I missed that during the review. Fixed in #17702

Reply:
Mistake in the correction lmao I need to focus more next time!